src/components/PromotionCard.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { RiTShirtLine, RiMotorbikeLine } from 'react-icons/ri';
+import { GiPerfumeBottle } from 'react-icons/gi';
+import { IconContext } from 'react-icons';
+
+export const PromotionCard = (props) => {
+    const { promotionsListData } = props;
+    // console.log("PromotionCard", promotionsListData);
+
+    if (promotionsListData.name === "beauty")
```

Comment:
This condition could be more simpler by returning the content of else directly if the condition is false.